sdk/lusid/utilities/__init__.py

```diff
@@ -0,0 +1,4 @@
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from lusid.utilities.api_configuration import ApiConfiguration
+from lusid.utilities.api_configuration_loader import ApiConfigurationLoader
+from lusid.utilities.refreshing_token import RefreshingToken
```

Comment:
The ApiConfigurationLoader takes "api_configuration" as a parameter. As it returns the ApiConfiguration class I think this is a bit confusing. I'd suggest renaming this to something like "api_secrets_filename" or something to that effect and make it clear in the doc string of the load function of ApiConfigurationLoader that it is a string.

Reply:
done